### ISSUE TITLE
Getting a TransportError(414, ....) when using scan

### ISSUE THREAD

**Issue by spott** [elastic/elasticsearch-py#1014]
It seems to be related to the very (*very*, *very*) long URI that is used for the pagination.  [this](https://github.com/elastic/elasticsearch-ruby/issues/423) bug in the elasticsearch-ruby repo seems to hint that the "scroll_id" parameter is better passed in the body of the request, but I don't know how to do that in elasticsearch-dsl.

I'm not doing anything complicated here, just doing a search with a single result:

```
s = ed.Search(using=client, index='myindex_*').query('match', _id="<some_id>")

for hit in s.scan():
    print(hit)
```

edit: As it is probably important: I'm using elasticsearch-py and elasticsearch-dsl-py versions 6.4.0

Let me know if any more information is needed.

**Comment by HonzaKral** [comment]
Transferred the issue here since it is really a problem with `elasticsearch-py` and not the dsl

**Comment by HonzaKral** [comment]
This has been fixed in a98fa8ba9f6b6c2aac1dc5dcbefd089c8834ce2c
